src/CONST.ts

```diff
@@ -703,7 +703,7 @@ const CONST = {
         TOOLTIP_SENSE: 1000,
         TRIE_INITIALIZATION: 'trie_initialization',
         COMMENT_LENGTH_DEBOUNCE_TIME: 500,
-        SEARCH_FOR_REPORTS_DEBOUNCE_TIME: 300,
+        SEARCH_OPTION_LIST_DEBOUNCE_TIME: 200,
```

Comment:
```suggestion
        SEARCH_OPTION_LIST_DEBOUNCE_TIME: 300,
```

I think we should keep this at 300ms

Reply:
updated to 300ms